outOfSyncNoti.sh

```diff
@@ -21,12 +29,10 @@ echo "latest block number from etherscan: $ethBlock"
 echo "Our current block number: $curBlock"
 
 # Check if node is out of sync.
-# Will send notification when current block number is <numberofblocksbehind> blocks behind latest block on etherscan.
-genieKey=$2
-numberofblocksbehind=${3:-200}
-lowestBlock=$(($ethBlock-$numberofblocksbehind))
-
+# Will send notification when current block number is <threshold> blocks behind latest block on etherscan.
+lowestBlock=$(($ethBlock-$threshold))
 if [ $curBlock -lt $lowestBlock ];then
```

Comment:
`if [ $(( ${ethBlock} - ${curBlock} )) -ge $threshold ]; then`
and remove misleading `lowestBlock`

Reply:
If I do this, I need to change the default from `-200` to `200` right?